axis_utils/include/axis_utils2.inc

```diff
@@ -351,119 +182,88 @@ end subroutine axis_edges
     !        lon =      0 1 2 3 4 5  ...  358 359; lon_strt = 3 ==>
     !        tranlon =  3 4 5 6 7 8  ...  359 360 361 362; istrt = 4
 
-    real, intent(inout), dimension(:) :: lon
-    real, intent(in) :: lon_start
-    integer, intent(out) :: istrt
+    real(kind=FMS_AU_KIND_), intent(inout), dimension(:) :: lon
+    real(kind=FMS_AU_KIND_), intent(in)                  :: lon_start
+    integer, intent(out)                                 :: istrt
 
 
-    integer :: len, i
-    real :: lon_strt, tmp(size(lon(:))-1)
+    integer                 :: len, i
+    real(kind=FMS_AU_KIND_) :: lon_strt, tmp(size(lon(:))-1)
 
     len = size(lon(:))
 
-    do i=1,len
+    do i = 1, len
        lon(i) = lon_in_range(lon(i),lon_start)
     enddo
 
-    istrt=0
-    do i=1,len-1
+    istrt = 0
+    do i = 1,len-1
        if (lon(i+1) < lon(i)) then
-          istrt=i+1
+          istrt = i+1
           exit
        endif
     enddo
 
     if (istrt>1) then ! grid is not monotonic
-       if (abs(lon(len)-lon(1)) < epsln) then
+       if (abs(lon(len)-lon(1)) < real(epsln, r8_kind)) then
```

Comment:
@mcallic2 is there a reason epsln is converted to r8?

Reply:
Module variable

Reply:
if epsln is a module variable declared as r8, should it be real(epsln, FMS_AI_KIND_)?

Reply:
You told me the opposite of that. You said that all module variables need to stay r8_kind throughout the module. I remember because we were having problems in monin_obukhov doing the one routine call

Reply:
module variables need to be declared as r8 and need to be appropriately converted/cast-ed to the appropriate precision.

Reply:
Got ya got ya, I prob misunderstood something. I fixed